lib/collection/src/operations/types.rs

```diff
@@ -17,6 +17,12 @@ use crate::wal::WalError;
 /// Type of vector in API
 pub type VectorType = Vec<VectorElementType>;
 
+#[derive(Debug, Deserialize, Serialize, JsonSchema)]
+pub enum PayloadContent {
+    Payload(TheMap<PayloadKeyType, PayloadType>),
+    Json(String)
+}
+
```

Comment:
`Json` option assumes and declares for the API schema, that any valid json is acceptable as a payload. So far this is not the case, so such a definition would be misleading.

Reply:
Moreover - return a json as string would require double parsing of the service response.